### ISSUE TITLE
Description Page adjustments

### ISSUE THREAD

**Issue by annachiaretta** [coyote-team/coyote#549]
Suggested updates to the description page layout to improve usability.

1. The description itself is not prominent at all, especially with alt text. Furthermore, it is unlabeled and quite close to the fields that indicate it's affiliated resource, author, status, etc. I argue the description should be labeled as such and foregrounded to its related information by text scale and/or spacing to separate it from its associated metadata.
2. Edit, Approve, Reject action buttons should be separated from Describe again, appearing below the description info block, not only for consistency around action placement on the page level, but also to show these actions more clearly as being related to the description rather than the resource.
3. Add "Approve and Next" button to easily move at the page level through ready to review descriptions.

screenshot of [a description page](https://staging.coyote.pics/organizations/1-museum-of-contemporary-art-chicago/representations/7803) shows suggested new placement of description action buttons with a pink circle:
<img width="1784" alt="Screen Shot 2021-02-25 at 13 52 38" src="https://user-images.githubusercontent.com/10015955/109203945-c29d5500-7772-11eb-9f0b-f735910d8499.png">

**Comment by sinabahram** [comment]
This is great stuff, Anna.

The third item, @flipsasser , we'll need to discuss RE getting the accessibility right.

@annachiaretta if I may request your alt text directly on the screenshots, that would rock, just so it doesn't put unlabeled images in the GitHub interface.

**Comment by flipsasser** [comment]
Thanks everyone! @sinabahram let's chat about the accessibility next week during our weekly sync

**Comment by flipsasser** [comment]
This has been implemented!
